run_sorting_hat_test_data.sample.sh

```diff
@@ -138,7 +150,7 @@ snakemake --use-conda --conda-frontend mamba \
         tomcat_host_password=$TOMCAT_HOST_PASSWORD \
         tomcat_host=$TOMCAT_HOST \
         prot_magetabfiles=$PROT_MAGETABFILES \
-        sm_options="--use-conda --conda-frontend mamba --keep-going $PROFILE_LINE -j $NJOBS $CONDA_PREFIX_LINE $FORCE_ALL --restart-times $RESTART_TIMES " \
+        sm_options="--slurm --use-conda --conda-frontend mamba --keep-going $PROFILE_LINE -j $NJOBS $CONDA_PREFIX_LINE $FORCE_ALL --restart-times $RESTART_TIMES " \
```

Comment:
```suggestion
        sm_options="$slurm --use-conda --conda-frontend mamba --keep-going $PROFILE_LINE -j $NJOBS $CONDA_PREFIX_LINE $FORCE_ALL --restart-times $RESTART_TIMES " \
```

Reply:
Looks good to me once above suggested added or adapted other way it should be good to go.. will approve once you add them